tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -1557,6 +1634,114 @@ describe("clipboard", () => {
     expect(getStyle(model, "C2")).toEqual({});
   });
 
+  test("copy cells with CF => remove origin CF => paste => it should paste with original CF", () => {
+    const model = new Model();
+    setCellContent(model, "A1", "1");
+    setCellContent(model, "A2", "0");
+    setCellContent(model, "A3", "1");
```

Comment:
The test doesn't need content in the cell to be relevant. Keep it short and minimal
```suggestion
```